scalding-core/src/main/scala/com/twitter/scalding/filecache/DistributedCacheFile.scala

```diff
@@ -111,8 +112,10 @@ final case class UncachedFile private[scalding] (source: Either[String, URI]) {
     def makeQualifiedURI(uri: URI, conf: Configuration): URI =
       makeQualified(new Path(uri.toString), conf) // uri.toString because hadoop 0.20.2 doesn't take a URI
 
-    def makeQualified(p: Path, conf: Configuration): URI =
-      p.makeQualified(p.getFileSystem(conf).getUri, p.getFileSystem(conf).getWorkingDirectory).toUri
+    def makeQualified(p: Path, conf: Configuration): URI = {
+      val fileSystem: FileSystem = p.getFileSystem(conf)
```

Comment:
note, we generally don't do a type annotation, especially if we need an import to do so, when it is not returned from a public method.